src/rust/ide/view/graph-editor/src/component/visualization/foreign/java_script/instance.rs

```diff
@@ -159,7 +159,9 @@ impl InstanceModel {
         let logger                        = Logger::new("Instance");
         let root_node                     = Self::create_root(scene,&logger)?;
         let (preprocessor_change,closure) = Self::preprocessor_change_callback();
-        let init_data                     = JsConsArgs::new(root_node.clone_ref(), closure);
+        let init_data                     = JsConsArgs::new(root_node.clone_ref(),
+                                                            StyleWatch::new(&scene.style_sheet),
+                                                            closure);
```

Comment:
distribute that to variables please. Also, in case it would be impossible somewhere else to distribute things to variables, we use 4 spaces indentation, not aligned with rest of args.